test/tablet_scripts/test_topology.py

```
@@ -0,0 +1,97 @@
+def test_resolve_table_id_rejects_ambiguous_bare_table_name() -> None:
+    topo = Topology()
+    table1 = UUID("11111111-1111-1111-1111-111111111111")
+    table2 = UUID("22222222-2222-2222-2222-222222222222")
+    topo._tables[table1] = ("ks1", "users")
+    topo._tables[table2] = ("ks2", "users")
+    topo._tablet_maps[table1] = TabletMap(table1)
+    topo._tablet_maps[table2] = TabletMap(table2)
+
+    try:
+        resolve_table_id(topo, "users")
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)